src/activity_actor.cpp

```diff
@@ -1646,7 +1646,7 @@ void unload_activity_actor::unload( Character &who, item_location &target )
         if( contained->ammo_type() == ammotype( "plutonium" ) ) {
             contained->charges /= PLUTONIUM_CHARGES;
         }
-        if( who.as_player()->add_or_drop_with_msg( *contained, true ) ) {
+        if( who.as_player()->add_or_drop_with_msg( *contained, true, &it ) ) {
```

Comment:
This avoids counting the volume of the item that is being unloaded as space the unloaded items can go to.